_includes/mailchimp.html

```diff
@@ -0,0 +1,18 @@
+<!-- Begin MailChimp Signup Form -->
+<div id="mc_embed_signup">
+	<form action="//thefirehoseproject.us11.list-manage.com/subscribe/post?u=48d53d72a28bb1c0fca326b5a&amp;id=953d41e871" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+    	<div id="mc_embed_signup_scroll">
+			<label for="mce-EMAIL">Want to be notified when new content is added?</label>
+			<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>
```

Comment:
Line 5 can be completely removed. Line 6 can be modified to have the "Want to be notified when new content is added?" in the placeholder. 

```
<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="Want to be notified when new content is added?" required>
```

From there you will have to do some work figuring out how to make thinks look like they belong. Look into the CSS bootstrap class "form-control", that will change the input box to make it look more like it belongs.